web/src/cypress/support/index.ts

```diff
@@ -14,6 +14,8 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
+import 'cypress-plugin-retries'
+Cypress.env('RETRIES', 2)
```

Comment:
is the `command.js` comment relevant/could be updated?

Reply:
Old comment from the original Cypress sample.ts. I'll remove it